initial.c

```diff
@@ -0,0 +1,13 @@
+#include <initials.c> /*Sheqi Zhang 2016/06/29*/
+
+main()
```

Comment:
You forgot to include "int" - it should be int main() - the code returns a warning without it.